src/python/pants/task/task.py

```diff
@@ -175,6 +176,7 @@ def __init__(self, *args, **kwargs):
     self.context = context
     self._workdir = workdir
 
+    self._task_name = type(self).__name__
```

Comment:
This should maybe/probably use `Task.stable_name`.

Reply:
Yeah, was keeping status quo here since this name plumbs through to stats via InvalidationReport. Changing would change the values in the emitted csv's `task_name` column. Although InvalidationReport is not `:API: public`, I was erring on the side of an emitted db like this being inherently public since pants itself doesn't use the data internally.